python/verticox/vantage6.py

```diff
@@ -0,0 +1,145 @@
+import time
+from typing import List
+
+import grpc
+import pandas as pd
+from vantage6.client import ContainerClient
+from vantage6.tools.util import info
+
+from verticox import datanode
+from verticox.aggregator import Aggregator
+from verticox.grpc.datanode_pb2_grpc import DataNodeStub
+
+PORT = 8888
+MAX_RETRIES = 20
+GRPC_OPTIONS = [('wait_for_ready', True)]
+SLEEP = 5
+DATANODE_TIMEOUT = 60 * 60
+DATA_LIMIT = 10
+DEFAULT_PRECISION = 1e-5
+
+
+def _limit_data(data):
+    return data.iloc[:5]
+
+
+def verticox(client: ContainerClient, data: pd.DataFrame, feature_columns: List[str],
+             event_times_column: str, event_happened_column: str, datanode_ids: List[int] = None,
+             precision: float = DEFAULT_PRECISION,
+             *_args, **_kwargs):
+    """
+    TODO: Describe precision parameter
```

Comment:
Looks like `precision` is the only parameter that is currently described. Describe the others while you still remember what they mean?